linenoise.hpp

```diff
@@ -1763,12 +1855,12 @@ inline void linenoiseBeep(void) {
  *
  * The state of the editing is encapsulated into the pointed linenoiseState
  * structure as described in the structure definition. */
-inline int completeLine(struct linenoiseState *ls, char *cbuf, int *c) {
+int linenoiseState::completeLine(char *cbuf, int *c) {
```

Comment:
If you remove `inline` keyword, we are not able to include the `linenoise.hpp` in more than one compilation unit. For example, if we include this file in `a.cpp` and `b.cpp` and link them to make an executable, the linker doesn't allow use to build.

Reply:
Got it - I pushed a commit adding them back in, as well as an example to build that tests that inclusion case.